frontend/src/assets/css/variables.css

```diff
@@ -0,0 +1,5 @@
+:root {
+  --text-light: #90a3bf;
```

Comment:
Looks like it our design colors.
![image](https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/78347072/6342566b-aaca-41b8-ba21-798ccf76942a)

We should add proper name for it.
Example: `bt-gray`
or use some tools for it - https://www.color-name.com/hex/90a3bf

Reply:
okay, thanks